src/js/components/ToggleButtonGroup/StyledToggleButtonGroup.js

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+import { Button } from '../Button';
+import { roundStyle, edgeStyle } from '../../utils';
+
+export const StyledButton = styled(Button)`
+  border-radius: ${(props) => props.theme.global.control.border.radius};
```

Comment:
```suggestion
  border-radius: 0;
```
I believe this should be 0 to baseline the border-radius, then props.round should just apply to the first/last item